Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -8999,5 +9068,20 @@ private void VerticalScrollBar_Scroll(object sender, ScrollEventArgs e)
         {
             ProcessVerticalScroll(e.ScrollEventType);
         }
+
+        private void LoadMoreDataFromIncrementalItemsSourceAsync()
+        {
+            if (IncrementalLoadingTrigger == IncrementalLoadingTrigger.Edge && DataConnection.IsDataSourceIncremental && DataConnection.HasMoreItems && !DataConnection.IsLoadingMoreItems)
+            {
+                var bottomScrollOffHeight = Math.Max(0, EdgedRowsHeightCalculated - CellsHeight - VerticalOffset);
```

Comment:
nit: bottomScrolledOffHeight

Reply:
LoadMoreDataFromIncrementalItemsSource() also needs to be called when DataGrid.CellsHeight changes.
So a LoadMoreDataFromIncrementalItemsSource() call should be added below the existing line
`    _rowsPresenterAvailableSize = value;
`
This is to ensure that when the DataGrid's height increases, an incremental load is triggered as needed (because the scrolled off height decreases).